### PR TITLE
Introduce modals for featured elements

### PR DESCRIPTION
**Pull request by dinuand**
Modals to be displayed on user-click actions with text and image. The rest of the site is blurred and the focus is on the modal. The modal can be closed and the user can see the website again.

### CODE REVIEW

**Comment by maria** [comment]
I will take a look. It seems you imported something, but it wasn't what I had in mind.

File: index.html [hunk 1]
```diff
@@ -22,6 +22,12 @@
     <link href="assets/css/stylesheet.css" rel="stylesheet">
     <script src="assets/js/core.js"></script>
 
+    <!-- Bootstrap modal -->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.1/css/bootstrap.min.css">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.1/css/bootstrap-theme.min.css">
```
Comment (maria) [inline index.html:27]: we already import bootstrap [above](https://github.com/dinuand/sites_template/blob/master/index.html#L11-L13)

File: index.html [hunk 2]
```diff
@@ -22,6 +22,12 @@
...
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.1/js/bootstrap.min.js"></script>
```
Comment (maria) [inline index.html:29]: https://github.com/rosedu/sites_template/pull/8/files#r23946729

File: index.html [hunk 3]
```diff
@@ -22,6 +22,12 @@
     <link href="assets/css/stylesheet.css" rel="stylesheet">
     <script src="assets/js/core.js"></script>
 
+    <!-- Bootstrap modal -->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.1/css/bootstrap.min.css">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.1/css/bootstrap-theme.min.css">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js"></script>
```
Comment (maria) [inline index.html:28]: We already import all jquery above.

File: index.html (hunk 2 above)
Comment (maria) [inline index.html:29]: Sorry, we only imported the css not the js.

**Comment by maria** [comment]
Nice work! I moved it here #9 so I can make some style changes.

File: assets/css/stylesheet.css [hunk 4]
```diff
@@ -223,7 +223,20 @@ a:hover {
...
+  width:500px;
+  margin: 100px auto;
+  background-color: #fff;
+  border:1px solid #000;
+  padding:15px;
+  text-align: center;
+  box-shadow: 0px 0px 8px 8px rgba(255,255,255,1);
+  text-shadow: 0px 0px 10px rgba(255, 255, 255, 0.9);
+  transform: scale(0.9);
+  opacity: 0.9;
```
Comment (maria) [inline assets/css/stylesheet.css:235]: All of this is what for nothing, since this element doesn't exist. :)

Comment (dinuand) [inline assets/css/stylesheet.css:235]: You're right, it does nothing. I left it just as an alternative for styling the modal pop-up window. I didn't know which way is more preferable.

**Pull request closed without merging**